docs/using-controller-manager-with-kubeadm.md

```diff
@@ -14,7 +14,9 @@ Then follow the usual steps to bootstrap the other nodes using `kubeadm join`
 
 Step 3: build the container image using the following (`make bootstrap` will download go SDK and glide if needed)
 ```
-make build-image
+export GOOS=linux
+make images
```

Comment:
`GOOS=linux make images` should be good here. then we don't have to do the export or unset

Reply:
Thanks, fixed in the latest commit.